app/routes/mock-login.js

```diff
@@ -8,11 +8,12 @@ export default class MockLoginRoute extends Route {
     },
   };
 
+  @service router;
   @service session;
   @service store;
 
   beforeModel() {
-    if (this.session.isAuthenticated) this.transitionTo('index');
+    if (this.session.isAuthenticated) this.router.transitionTo('index');
```

Comment:
Unrelated to this PR, but might be a nice follow up:

I think modern ember-simple-auth has some util methods for this. In this case I think you could replace this with `prohibitAuthentication`: https://ember-simple-auth.com/api/classes/SessionService.html#method_prohibitAuthentication There is an equivalent for the inverse as well.